### ISSUE TITLE
Alejandro Marquez Cervantes

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#65]

**Comment by aflies1** [comment]
Resume - https://drive.google.com/drive/folders/1px1YjR8lq2-6MiFnEm3wj_t3X06C9cBC?usp=sharing

**Comment by albertoleon7794** [comment]
Answers from interview: https://docs.google.com/document/d/1aOLwidqtlZmITv-AuuAAxqNZZPf-F6NJzeXbDMgO9wE/edit?usp=sharing

**Comment by albertoleon7794** [comment]
Score: https://docs.google.com/spreadsheets/d/11WCytb3tYyet0f4IJ0npFTnldDtFrxL0Mx9M8MzaZ0Y/edit?usp=sharing

**Comment by albertoleon7794** [comment]
Notes: 

- Candidate has great soft skills and is very open and honest. 
- Great communicator. 
- Passionate about learning. 
- Very low SQL knowledge.
I need go over his code assessment especially the SQL to verify his skill.

**Comment by ldaudet** [comment]
Alejandro Marquez
    Do they have a strong understanding of Java, can they pick things up quickly

He started coding from 11, and really started to learn java when he was 16, and then got deeper into it when he got in college at 17. Each of his classes had about 3 small week project and 1 month long project to be completed on the side.

He has a job teaching java and python to kids on the weekend. He made a machine learning algo to recognise hand written digit own his own. His most complex project was the voice controlled car. He likes to work on his own project in Java because of the declarative nature of things. He likes to think that he is a very organized person, always trying to hold himself up to deadlines etc... He likes to start a new project by mapping it out quickly and then starting the work itself. 

He likes team members that like to offer ideas, and challenging him rather than being yes sayers. He doesn't like people that work all by themselves and aren't team player. 

    Can they learn quickly on the fly? Schwab isn't going to expect them to know everything but the faster they pick things up the better. This team will start slow with some QA and get up to speed on their overall project and codebase before we start to take on dev tasks. (they will be working and integrated into their AGILE/SCRUM)

When faced with a new issue he has no idea about, he will first ticker with it, mess up line by line to figure it out. Says he's really good at googling stuff, which is a MASSIVE plus. Will reach out to a manager if really stuck when he feels that the efficiency would be greater if asking a manager for example.

    Great communication - if they don't know something we'd want them to try and figure it out on their own, but ask our manager and senior devs for help. This is a 12 month project so it will be important for them to build strong communication/relationships with the team and Schwab's devs.


Not used SCRUM before. Said that he never really learned to talk to people before college, and got to learn how to do so during his customer service experience. Has a passion for teaching, which he made come true with his second work experience at AiGo learning where he teaches kids every weekend. He also did math videos for another company before but didn't like the way they work.


I asked him what a singleton class was and he didn't know the answer (hadn't checked it since talking to alberto). He says at the end that he wanted to add that he was a very creative person.

He asked at the end how he compared to the other applicants by asking what I had seen some of them have that he didn't, which I thought was pretty good. I usually think that asking the interviewer to compare you to other applicants shows some insecurities, but the way he phrased it only looking for ways to improve I thought was pretty slick.
